### PR TITLE
Remove python support from piksi_firmware setup

### PR DESCRIPTION
**Pull request by mfine**
Python stuff is now in piksi_tools - pull the support for setting it up from the scripts.

/cc @denniszollo @mookerji

### CODE REVIEW

**Comment by mfine** [comment]
Transferal of setup to piksi_tools - see https://github.com/swift-nav/piksi_tools/pull/41. In support of https://github.com/swift-nav/piksi_tools/issues/40.

File: setup/ansible/roles/common_ubuntu/tasks/sys_deps.yml [hunk 1]
```diff
@@ -15,45 +14,13 @@
...
     - git
     - build-essential
-    - python
-    - python-setuptools
-    - python-pip
-    - python-virtualenv
     - cmake
-    - swig
     - wget
-    - cmake
```
Comment (cbeighley) [inline setup/ansible/roles/common_ubuntu/tasks/sys_deps.yml]: cmake is still required for building libswiftnav - is that handled by something in the libswiftnav build dependencies in the provisioning?

Comment (mfine) [inline setup/ansible/roles/common_ubuntu/tasks/sys_deps.yml]: There's two cmake's there - I pulled the extra one.

Comment (mfine) [inline setup/ansible/roles/common_ubuntu/tasks/sys_deps.yml]: screen shot 2015-05-06 at 11 17 16 am

That extra one is not meaningful, right?

**Comment by cbeighley** [comment]
Installation section of README.md still talks about using Piksi Console, we should remove that and maybe note that now provisioning only installs piksi_firmware dependencies?

**Comment by mfine** [comment]
Updated the readme - leaving in the reference to the console and the binary location.

File: setup/ansible/roles/common_ubuntu/tasks/sys_deps.yml (hunk 1 above)
Comment (mookerji) [inline setup/ansible/roles/common_ubuntu/tasks/sys_deps.yml]: Right.

**Comment by cbeighley** [comment]
@mookerji okay to merge?

**Comment by mookerji** [comment]
lgtm

**Pull request merged**